sw/device/lib/dif/dif_spi_device_unittest.cc

```diff
@@ -1829,28 +1821,6 @@ TEST_F(TpmTest, CommandAndData) {
   EXPECT_READ32(SPI_DEVICE_TPM_STATUS_REG_OFFSET,
                 {
                     {SPI_DEVICE_TPM_STATUS_CMDADDR_NOTEMPTY_BIT, 0},
-                    {SPI_DEVICE_TPM_STATUS_RDFIFO_NOTEMPTY_BIT, 1},
-                    {SPI_DEVICE_TPM_STATUS_RDFIFO_DEPTH_OFFSET, 14},
-                    {SPI_DEVICE_TPM_STATUS_WRFIFO_DEPTH_OFFSET, 4},
-                });
-  EXPECT_EQ(dif_spi_device_tpm_write_data(&spi_, /*length=*/9, data),
-            kDifOutOfRange);
-
-  EXPECT_READ32(SPI_DEVICE_TPM_STATUS_REG_OFFSET,
-                {
-                    {SPI_DEVICE_TPM_STATUS_CMDADDR_NOTEMPTY_BIT, 0},
-                    {SPI_DEVICE_TPM_STATUS_RDFIFO_NOTEMPTY_BIT, 1},
-                    {SPI_DEVICE_TPM_STATUS_RDFIFO_DEPTH_OFFSET, 15},
-                    {SPI_DEVICE_TPM_STATUS_WRFIFO_DEPTH_OFFSET, 4},
-                });
-  EXPECT_EQ(dif_spi_device_tpm_write_data(&spi_, /*length=*/5, data),
-            kDifOutOfRange);
-
-  EXPECT_READ32(SPI_DEVICE_TPM_STATUS_REG_OFFSET,
-                {
-                    {SPI_DEVICE_TPM_STATUS_CMDADDR_NOTEMPTY_BIT, 0},
-                    {SPI_DEVICE_TPM_STATUS_RDFIFO_NOTEMPTY_BIT, 1},
-                    {SPI_DEVICE_TPM_STATUS_RDFIFO_DEPTH_OFFSET, 1},
```

Comment:
@a-will I removed these checkers as DIF has been changed to check only if SW writes more than the FIFO size. Please take a look.